h-youjin/머쓱이보다-키-큰-사람.js

```diff
@@ -0,0 +1,5 @@
+function solution(array, height) {
+  const answer = 0;
+  answer = array.filter((v) => v > height).length;
+  return answer;
```

Comment:
```suggestion
  const answer = array.filter((v) => v > height).length;
  return answer;
```
- const 로 선언하고 재할당 할 수 없으므로 filter된 배열을 바로 할당해줬습니다.

Reply:
네렛!!!